x-pack/test/observability_functional/apps/observability/alerts/pagination.ts

```diff
@@ -34,7 +34,8 @@ export default ({ getService }: FtrProviderContext) => {
       await esArchiver.unload('x-pack/test/functional/es_archives/infra/metrics_and_logs');
     });
 
-    describe(`When less than ${ROWS_NEEDED_FOR_PAGINATION} alerts are found`, () => {
+    // This will fail after removing workflow filter i.e. show all not only "open"  https://github.com/elastic/kibana/issues/119946
+    describe.skip(`When less than ${ROWS_NEEDED_FOR_PAGINATION} alerts are found`, () => {
```

Comment:
I don't want to `skip` this. But after showing all the alerts (not only with `open` workflow), the archiver data will display more than 8 active alerts.

Any suggestion?

Reply:
@fkanout How about you use:

`await observability.alerts.common.setAlertStatusFilter(ALERT_STATUS_RECOVERED);` instead?

Will this show you less than ROWS_NEEDED_FOR_PAGINATION (10) alerts?

Reply:
Recovered ones are more than the active ones 

>const ACTIVE_ALERTS_CELL_COUNT = 78;
const RECOVERED_ALERTS_CELL_COUNT = 150;